src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java

```diff
@@ -0,0 +1,414 @@
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagInt;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.*;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.World;
+import org.apache.commons.lang3.tuple.Pair;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static codechicken.lib.raytracer.RayTracer.*;
+import static gregtech.api.render.Textures.CLIPBOARD_RENDERER;
+import static gregtech.common.items.MetaItems.CLIPBOARD;
+
+public class MetaTileEntityClipboard extends MetaTileEntity implements IRenderMetaTileEntity, IFastRenderMetaTileEntity {
+    private static final AxisAlignedBB CLIPBOARD_AABB = new AxisAlignedBB(2.75 / 16.0, 0.0, 0.0, 13.25 / 16.0, 1.0, 0.4 / 16.0);
+    public static final float scale = 1;
+    public FakeModularGui guiCache;
+    public FakeModularUIContainerClipboard guiContainerCache;
+    private static final Cuboid6 pageBox = new Cuboid6(3 / 16.0, 0.25 / 16.0, 0.25 / 16.0, 13 / 16.0, 14.25 / 16.0, 0.3 / 16.0);
+    private boolean isClientReceivingUpdates = false;
+
+    private static final int RENDER_PASS_NORMAL = 0;
+    private static final NBTBase NO_CLIPBOARD_SIG = new NBTTagInt(0);
+
+
+    public MetaTileEntityClipboard(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId);
+    }
+
+    @Override
+    public void update() {
+        super.update();
+        if (this.getWorld().isRemote) {
+            if (guiCache != null)
+                guiCache.updateScreen();
+        } else {
+            if (!isClientReceivingUpdates && getOffsetTimer() % 20 == 0)
+                createFakeGui();
+            if (guiContainerCache != null)
+                guiContainerCache.detectAndSendChanges();
+        }
+    }
+
+    @Override
+    public int getLightOpacity() {
+        return 0;
+    }
+
+    @Override
+    public void renderMetaTileEntityDynamic(double x, double y, double z, float partialTicks) {
+        if (this.getClipboard() != null)
+            CLIPBOARD_RENDERER.renderGUI(x, y, z, this.getFrontFacing(), this, partialTicks);
+    }
+
+    @Override
+    public void renderMetaTileEntityFast(CCRenderState renderState, Matrix4 translation, float partialTicks) {
+        CLIPBOARD_RENDERER.renderBoard(renderState, translation.copy(), new IVertexOperation[]{}, getFrontFacing(), this, partialTicks);
+    }
+
+    public AxisAlignedBB getRenderBoundingBox() {
+        return new AxisAlignedBB(getPos().add(-1, 0, -1), getPos().add(2, 2, 2));
+    }
+
+    @Override
+    public boolean shouldRenderInPass(int pass) {
+        return pass == RENDER_PASS_NORMAL;
+    }
+
+    @Override
+    public boolean isGlobalRenderer() {
+        return true;
+    }
```

Comment:
false plz, i don't think it l large enough that need to render globally, unless your collision box is wrong